apps/courses/src/components/RichTextEditor.tsx

```diff
@@ -88,6 +96,49 @@ const withLinks = (editor) => {
   return editor;
 };
 
+const useStickyToolbar = () => {
```

Comment:
I'm honestly surprised we cannot use `position: sticky` for this. It's available in CSS in all browsers that we support. Why did we need to implement a Javascript solution to this?